homeassistant/components/verisure.py

```diff
@@ -53,6 +54,8 @@
         vol.Optional(CONF_SMARTPLUGS, default=True): cv.boolean,
         vol.Optional(CONF_THERMOMETERS, default=True): cv.boolean,
         vol.Optional(CONF_SMARTCAM, default=True): cv.boolean,
+        vol.Optional(CONF_POLLING_RATE, default=1): vol.All(
```

Comment:
Normally we call this `CONF_SCAN_INTERVAL`.

Reply:
Ok. I should change that. When is the next release?

Reply:
In ~7 days.

Reply:
Great., I will be able to update during the weekend. Thanks @MartinHjelmare